wp-content/themes/core/components/card/css/card.pcss

```diff
@@ -60,6 +82,23 @@
 	.c-card--style-outlined & {
 		margin-top: calc(-1 * var(--spacer-30));
 	}
+
+	/* CASE: Style: Inline */
+	.c-card--style-inline & {
+		display: none;
+		flex-shrink: 0;
+
+		@media(--viewport-medium) {
+			display: block;
+			width: 152px;
```

Comment:
Does this align to a grid width? ANy chance we can use a grid width variable? Or, if not, can you document why it's specific? Same with line 99 below.